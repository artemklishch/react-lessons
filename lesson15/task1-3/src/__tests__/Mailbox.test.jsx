import React from 'react';
import Mailbox from '../Mailbox';
import { shallow } from 'enzyme';

describe('<Mailbox />', () => {
  it ('should get number of unread messages', () => {
    const props = {
      unreadMessages: [1,2,3,4,5,6,7,8,9,0],
    }
    const wrappedComponent = shallow(<Mailbox {...props} />);
    expect(wrappedComponent).toMatchSnapshot();
  })
});