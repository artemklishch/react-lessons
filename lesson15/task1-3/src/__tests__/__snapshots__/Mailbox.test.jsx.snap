// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Mailbox /> should get number of unread messages 1`] = `
<div
  className="mailbox"
>
  <span
    className="mailbox__text"
  >
    Messages
  </span>
  <span
    className="mailbox__count"
  >
    10
  </span>
</div>
`;

exports[`<Mailbox /> should get number of unread messages 2`] = `
<div
  className="mailbox"
>
  <span
    className="mailbox__text"
  >
    Messages
  </span>
</div>
`;
